tests/integration/test_uofm_process.py

```diff
@@ -0,0 +1,75 @@
+import pytest
+import boto3
+import json
+import os
+from load_env import load_env_file
+from processes import UofMProcess
+from managers import DBManager
+from model import Record
+
+class TestUofMProcess:
+    @pytest.fixture
+    def test_instance(self):
+        load_env_file('local', file_string='config/local.yaml')
+        return UofMProcess('complete', None, None, None, None, None, [])
+    
+    @pytest.fixture
+    def test_instance_QA(self):
+        load_env_file('local-qa', file_string='config/local-qa.yaml')
+    
+    def test_runProcess(self, test_instance_QA, test_instance: UofMProcess):
+        # Run process for only 1 record
+        # TODO - create S3 client and upload PDF
```

Comment:
we can remove the comments that have been done!

Reply:
Sure